playbook/workshops/twilio/twilio-basic.js/README.md

```diff
@@ -0,0 +1,7 @@
+# twilio-basic.js
+
+```
+uglifyjs src/js/loader.js src/lib/dweet.min.js src/js/string_utils.js \
```

Comment:
Should probably say this is a command that builds this. Or you could imply that, by prepending a `$`.